app/scripts/controllers/builds.js

```diff
@@ -20,6 +30,9 @@ angular.module('openshiftConsole')
 
     var buildConfigForBuild = $filter('buildConfigForBuild');
 
+    $scope.buildsVersion = APIService.getPreferredVersion('builds');
+    $scope.buildConfigsVersion = APIService.getPreferredVersion('buildconfigs');
```

Comment:
Let's not use `$scope` vars if these aren't used in the view

Reply:
sounds good.  kept that for consistency with other views, but agree, no point in blowing up `$scope`.